docs/getting-started-guides/docker-multinode.md

```diff
@@ -74,10 +74,11 @@ it is still useful to use containers for deployment and management, so we create
 You can specify the version on every node before install:
 
 ```sh
-export K8S_VERSION=<your_k8s_version (e.g. 1.2.0-alpha.6)>
+export K8S_VERSION=<your_k8s_version (e.g. 1.2.0-alpha.7)>
```

Comment:
BTW, do you know why `https://console.cloud.google.com/storage/kubernetes-release/release/latest.txt` requires authentication?
It would be a great way of getting the default in the future